components/entity-access-rules/rest/src/main/java/org/phenotips/data/permissions/rest/internal/NameAndEmailExtractor.java

```diff
@@ -109,7 +109,7 @@
         if (groupObject != null) {
             email = groupObject.getStringValue("contact");
         }
-        String name = document.getDocumentReference().getName();
+        String name = document.getTitle();
```

Comment:
What if the `title` field is missing? `getTitle()` does not default to the document name, while `getDisplayTitle()` does.

Reply:
`getDisplayTitle` (and `getPlainTitle`) only exist in the scriptable `Document` API, not in `XWikiDocument`. But yes, the problem is real.